Week2/homework/map-filter.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
-function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
-}
+const doubleOddNumbers = numbers => {
+  let newNumbers = [];
+  newNumbers = numbers.filter(number => number % 2 !== 0);
+  newNumbers = newNumbers.map(x => x * 2);
+  return newNumbers;
+};
```

Comment:
Ah - you can also chain the two methods `filter` and `map` together and rewrite this code as: 

```
const newNumbers = numbers.filter(number => number % 2 !== 0).map(x => x * 2);
return newNumbers;
```

Reply:
ahh yeah, I kinda forgot updating it since I did this one first and learned it later 😅 
will update ^^